src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -5,10 +5,176 @@
+        + "module code.\n\n"
+        + "\t\t\t\tExample(s):\tdelete cs1231\n\n";
+    public static final String EDIT_COMMAND = "To edit a module:\tType \"edit\" followed by a valid module "
+        + "code and the fields\n"
+        + "\t\t\t\tyou wish to edit (at least 1 field must be provided). You can\n"
+        + "\t\t\t\tonly edit the module code and tags of a module by using the\n"
+        + "\t\t\t\tprefixes \"c/\" and \"t/\" respectively.\n\n"
+        + "\t\t\t\tNote:\n" + "\t\t\t\tEditing tags replaces existing tags with new ones instead of\n"
+        + "\t\t\t\tchanging their respective descriptions. You can remove tags of a\n"
+        + "\t\t\t\tmodule simply by including the aforementioned prefix without a\n"
+        + "\t\t\t\tdescription as one of the edit fields, i.e., \"t/\".\n\n"
+        + "\t\t\t\tExample(s):\tedit cs1231 c/cs2100\n"
+        + "\t\t\t\t\t\t\tedit cs1231 c/cs2100 t/Foundation\n"
+        + "\t\t\t\t\t\t\tedit cs1231 t/ (to remove tags)\n\n";
+    public static final String FIND_COMMAND = "To find a" + "\t\t\tType \"find\" followed by the respective "
+        + "keywords or tags.\n"
+        + "specific module" + "\t\t(You can specify multiple keywords or tags)\n"
+        + "or a group of\n"
+        + "modules with" + "\t\tNote:\n"
+        + "common tags" + "\t\tKeywords can be specified as a sequence of characters of a\n"
+        + "or keywords:" + "\t\tmodule code whereas tags must be specified fully.\n\n"
+        + "\t\t\t\tExample(s):\tfind cs1231\n"
+        + "\t\t\t\t\t\t\tfind cs st (to find modules which module codes\n"
+        + "\t\t\t\t\t\t\tcontain \"cs\" or \"st\")\n"
+        + "\t\t\t\t\t\t\tfind foundation (to find modules with\n"
+        + "\t\t\t\t\t\t\t\"foundation\" as a tag)\n\n";
+    public static final String SEARCH_COMMAND = "To view details" + "\t\tType \"search\" followed by a "
+        + "valid module code.\n"
+        + "of a module:\n"
+        + "\t\t\t\tExample(s):\tsearch cs1231\n\n";
+    public static final String REQUIRED_COMMAND = "To view all" + "\t\t\tType \"required\".\n"
+        + "your required\nmodules:\n\n";
+    public static final String CHECKMC_COMMAND = "To check\t\t\tType \"checkmc\".\n"
+        + "your total MCs:\n\n";
+    public static final String TAG_COMMAND = "To view all" + "\t\t\tType \"tags\"\n"
+        + "your tags:\n\n";
+    public static final String LIST_COMMAND = "To list all" + "\t\t\tType \"list\".\n"
+        + "your completed\nmodules:\n\n";
+    public static final String CLEAR_COMMAND = "To clear GradPad:\tType \"clear\"\n\n";
+    public static final String EXIT_COMMAND = "To exit GradPad:\t\tType \"exit\".\n\n";
+
+    public static final String SHOWING_HELP_MESSAGE = ADD_COMMAND + DELETE_COMMAND + EDIT_COMMAND
+        + FIND_COMMAND + SEARCH_COMMAND + REQUIRED_COMMAND + CHECKMC_COMMAND + TAG_COMMAND + LIST_COMMAND
+        + CLEAR_COMMAND + EXIT_COMMAND;
```

Comment:
Will have to include SCIENCE_COMMAND and GEM_COMMAND as well

Reply:
aight bro will add in!